.github/workflows/release-test.yml

```diff
@@ -0,0 +1,28 @@
+name: Release tests
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  # trigger oqs-provider release tests on a tracker branch
+  oqs-provider-release-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger oqs-provider release tests
+        run: |
+          curl --silent \
+               --write-out "\n%{response_code}\n" \
+               --request POST \
+               --header "Accept: application/vnd.github+json" \
+               --header "Authorization: Bearer ${{ secrets.OQSBOT_GITHUB_ACTIONS }}" \
```

Comment:
[This secret doesn't seem to be set yet](https://github.com/open-quantum-safe/liboqs/settings/secrets/actions).

Reply:
Indeed not, I don't have sufficient permissions to add it (or view the link). @dstebila created an `oqsbot` PAT for CircleCI some time ago; we could reuse that if it's saved in cleartext anywhere. Otherwise, somebody with perms will need to generate a new one and add it.

Reply:
Done. CI passes now. @bhess please review this PR and re-base #1626 after merge: When pushing a commit suitably labelled ("... [trigger downstream]"), CI should turn green on https://github.com/open-quantum-safe/oqs-provider/pull/348

Reply:
Thanks, will do after merge.